source/faq.txt

```diff
@@ -156,18 +154,19 @@ avoid closing.
 
 One message every 3000 milliseconds is not enough to keep the sockets
 active, so several of the sockets will time out after 5000 milliseconds.
-Reduce the ``poolSize`` in the connection settings to fix the problem.
+Reduce the ``maxPoolSize`` option in the connection settings to lower
+the number of connection the driver can maintain in the connection
+pool.
 
-To specify the optional ``poolSize`` setting for your ``MongoClient``, declare
+To specify the optional ``maxPoolSize`` setting for your ``MongoClient``, declare
 it in the ``options`` object of the constructor as follows:
 
 .. code-block:: javascript
 
    const client = new MongoClient(uri, {
-     poolSize: <integer value>,
+     maxPoolSize: <integer value>,
```

Comment:
Question: does this change in the setting name apply to all versions of the driver that we document? If it applies to a different set of versions than the other changes, perhaps a different PR would work better. Alternatively, maybe setting a source constant for this with a value that differs based on version might work too.

Reply:
Yes, since Node v3.5 this is the correct name for this option